fem/bilinearform_ext.hpp

```diff
@@ -58,7 +58,7 @@ class BilinearFormExtension : public Operator
    virtual void Update() = 0;
 };
 
-/// Data and methods for fully-assembled bilinear forms
+/// Data and methods for fully-assembled bilinear forms NOT IMPLEMENTED HERE
```

Comment:
To be honest these comments confuse me.  Are these methods implemented somewhere else?  If so, where?  If not, perhaps we should say "NOT YET IMPLEMENTED".